android/app/src/main/java/io/numbersprotocol/capturelite/MainActivity.java

```diff
@@ -8,8 +8,39 @@
 import com.equimaps.capacitorblobwriter.BlobWriter;
 
 import java.util.ArrayList;
+import android.content.res.Configuration;
+import android.webkit.WebSettings;
 
 public class MainActivity extends BridgeActivity {
+  void setDarkMode() {
+    // WORKAROUND: Android device doesn't support media query for prefers-color-scheme
+    // @reference: https://github.com/ionic-team/capacitor/discussions/1978#discussioncomment-708439
+    int nightModeFlags = getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK;
+    WebSettings webSettings = this.bridge.getWebView().getSettings();
+    if (nightModeFlags == Configuration.UI_MODE_NIGHT_YES) {
+      if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.Q) {
+        // As of Android 10, you can simply force the dark mode
+        webSettings.setForceDark(WebSettings.FORCE_DARK_ON);
+      }
+    } else {
+      if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.Q) {
+        webSettings.setForceDark(WebSettings.FORCE_DARK_OFF);
+      }
+    }
+  }
```

Comment:
```suggestion
    final int nightModeFlags = getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK;
    WebSettings webSettings = this.bridge.getWebView().getSettings();
    if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.Q) {
        final int forceDark;
        forceDark = nightModeFlags == Configuration.UI_MODE_NIGHT_YES
                ? WebSettings.FORCE_DARK_ON
                : WebSettings.FORCE_DARK_OFF;
        webSettings.setForceDark(forceDark);
    }
```

The readability might be better this way. (Repeating the conditional statement looks a bit confusing)